Dockerfile

```diff
@@ -3,8 +3,22 @@
 #
 FROM alpine:3.10.1 as python
 
+## Sadly, not ready for Python 3 yet. Crudini and gcloud SDK not updated.
+## When we are ready, changes below ready to go
+## Package pruning
+##   python and python-dev: replace with python3 and python3-dev
+##   py-pip: only needed for python 2
+##   py-virtualenv: replaced with built-in venv in Python 3
+#RUN sed -i 's|http://dl-cdn.alpinelinux.org|https://alpine.global.ssl.fastly.net|g' /etc/apk/repositories
```

Comment:
Why commented out?

Reply:
Maybe add python 3 stuff to comments of PR somewhere for your own records. I don't want to keep code commented out in the `Dockerfile` as it's just techdebt.

Reply:
These comments are what we should run when we switch to Python3. I left them there as documentation as it took me a while to figure out. 

I commented them out because it is not reasonable for us to switch to only Python 3 at this time, as neither `gcloud sdk` nor `crudini` support Python 3 at this time.

Reply:
There's no imminent date as to when that will be. Copy & paste it to the PR as a comment for reference. Python3 is like Perl6. One day it will happen, just we don't know when. =)

(also, another reason I wish we didn't need the python dependencies altogether)